build.gradle

```diff
@@ -735,3 +735,73 @@ task distRpm(dependsOn: "jpackageImage", type: Rpm) {
 	prefix '/opt' // Relocations: /opt
 	arch = 'x86_64'
 }
+
+task distRpmInDocker {
+	group = "RH8 RPM build container"
+	String image = "openjdk:17-oraclelinux8"
+	String rh8ContainerName = "oracle-linux8"
+	doFirst {
+		def UID = "id -u".execute()
+		UID.waitFor()
+		UID = UID.text.trim()
+
+		def GID = "id -g".execute()
+		GID.waitFor()
+		GID = GID.text.trim()
+
+		//create rh8 docker container
+		exec {
+			executable "docker"
+			args "create", "--name", rh8ContainerName, "-e", "UID=${UID}", "-e", "GID=${UID}", "-it", "-v", "${projectDir}:/source/anet", image
+		}
+		//start rh8 container
+		exec {
+			executable "docker"
+			args "start", rh8ContainerName
+		}
+		//add user to container
+		exec {
+			executable "docker"
+			args "exec", rh8ContainerName, "sh", "-c", "groupadd -g ${GID} anetgroup ; useradd -u ${UID} -g ${GID} anetuser "
+		}
+		//install yum and Development tools
+		exec {
+			executable "docker"
+			args "exec", rh8ContainerName, "sh", "-c", "microdnf -y install git yum ; yum -y groupinstall \"Development Tools\" "
+		}
+		//clear caches and copy anet to home/anetuser/anet folder
+		exec {
+			executable "docker"
+			args "exec", "-u", "anetuser", rh8ContainerName, "sh", "-c", "cd /source/anet ; rm -rf client/node_modules/; rm -rf build/*;" +
+					" rm -rf .gradle/caches/; rm -rf ~/.gradle/caches/; cp -r /source/anet/ ~/"
+		}
+		//build rpm in docker and copy rpm to build folder
+		exec {
+			executable "docker"
+			args "exec", "-u", "anetuser", rh8ContainerName, "sh", "-c", "cd ~/anet; ./gradlew --stop; ./gradlew yarn_cache_clean; ./gradlew clean distRpm;" +
+					"cp -r build/ /source/anet/; "
+		}
```

Comment:
Suggestion to combine/simplify these commands to:
```
		// clean and build zip & rpm in docker
		exec {
			executable "docker"
			args "exec", "-u", "anetuser", rh8ContainerName, "sh", "-c", "cd /source/anet ;  rm -rf client/node_modules/ ;" +
				" env YARN_CACHE_FOLDER=${cacheDir} ./gradlew ${gradleOptions} clean distZip distRpm"
		}
```
so as not to copy things around unnecessarily, not having to remove existing caches, and also build the ZIP since we want to attach that to releases too.

Reply:
done